### PR TITLE
NCS v2.5.0 rebase

### PR DESCRIPTION
**Pull request by cvinayak**
Rebase to upstream example-application, https://github.com/zephyrproject-rtos/example-application/commit/c2ed36f007f9ce51d4ab5b19c51850989e7ffb7c

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,25 +29,25 @@ points to the development branch of Zephyr, also `main`.
...
 
-## Getting Started
+## Getting started
 
-Before getting started, make sure you have a proper Zephyr development
-environment. Follow the official
-[Zephyr Getting Started Guide](https://docs.zephyrproject.org/latest/getting_started/index.html).
+Before getting started, make sure you have a proper nRF Connect SDK development environment.
+Follow the official
+[Getting started guide](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/getting_started.html).
```
Comment (peknis) [inline README.md:39]:
```suggestion
[Installation guide](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/getting_started.html).
```
We do not have Getting started anymore. We have a redirect, but you could use https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/installation.html, instead.

File: README.md [hunk 2]
```diff
@@ -15,12 +15,12 @@ applications. Some of the features demonstrated in this example are:
...
-This repository is versioned together with the [Zephyr main tree][zephyr]. This
-means that every time that Zephyr is tagged, this repository is tagged as well
+This repository is versioned together with the [nRF Connect SDK main tree][sdk-nrf]. This
+means that every time that nRF Connect SDK is tagged, this repository is tagged as well
 with the same version number, and the [manifest](west.yml) entry for `zephyr`
-will point to the corresponding Zephyr tag. For example, the `example-application`
-v2.6.0 will point to Zephyr v2.6.0. Note that the `main` branch always
-points to the development branch of Zephyr, also `main`.
+will point to the corresponding nRF Connect SDK tag. For example, the `ncs-example-application`
+v2.3.0 will point to nRF Connect SDK v2.3.0. Note that the `main` branch always
```
Comment (peknis) [inline README.md:22]: Why not 2.5.0?

**Review by peknis — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (cvinayak) [inline README.md:22]: @peknis should the version be updated or it can stay as-is as it is a example sentence?

Comment (gmarull) [reply README.md:22]: note this is just an example, doesn't need to be kept up to date

Comment (peknis) [reply README.md:22]: I think we can have it as an example, but could also use v2.5.0. Haven't really been working with this doc before.

**Review by peknis — APPROVED** [review]

**Review by gmarull — APPROVED** [review]

**Comment by gmarull** [comment]
@cvinayak b13052c4970800b4e3ad0d6c240182992bb3e410 needs to be reverted, required changes did not make into our sdk-zephyr fork yet.

**Pull request merged**